docs/explanations/audit-scanner/policy-reports.md

```diff
@@ -35,12 +35,33 @@ evaluation (pass, fail, or skip), and a timestamp indicating when the
 evaluation took place. Additionally, you can optionally define severity and
 category annotations for your policies.
 
-You can also leverage the optional UI provided by the
-[policy-reporter](https://github.com/kyverno/policy-reporter) open source
+Operators can access the reports via ordinary `kubectl` commands. They can also
+leverage the optional UI provided by the
+[policy-reporter](https://kyverno.github.io/policy-reporter) open source
 project for monitoring and observability of the PolicyReport CRDs.
-Furthermore, operators can access the reports via ordinary `kubectl` commands.
 
-Let's take a look at some example audit results generated by the audit scanner:
+## Policy Reporter UI
+
+The Policy Reporter is shipped as a subchart of `kubewarden-controller`, refer
+to the [Audit Scanner Installation](../../howtos/audit-scanner) page for more
+info.
+
+The Policy Reporter UI provides a dashboard showcasing all violations from
+`PolicyReports` and the `ClusterPolicyReport`. See the following example:
+
+![Policy Reporter dashboard example](/img/policy-reporter_dashboard.png)
+
+In addition, it provides a tab for PolicyReports, and a tab for
+ClusterPolicyReports, with expanded info. See the following example of the
+PolicyReport results:
+
+![Policy Reporter PolicyReports example](/img/policy-reporter_policyreports.png)
+
+Additional features of Policy Reporter include forwarding of results to
+different clients (like Grafana Loki, Elasticsearch, chat applications),
+metrics endpoint, etc. Please refer to the
+[policy-reporter](https://kyverno.github.io/policy-reporter) docs for more
+info.
```

Comment:
Will we support all these features? If that's not the case, we should mentioned that if the user enabled these features, we cannot help fixing bugs in these components.

Reply:
Argh, added a new commit: https://github.com/kubewarden/docs/commit/225689e4eab4419d41fce0bab025f25f108e7a49 with "community docs" instead of docs, for clarification.

Sadly, when pushing I pushed directly to docs' `main` branch instead of my fork, sorry.
Please raise any review issue, I can open a follow-up PR if needed.